task2-bash/guessanumber.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "Loading game..."
+tar xf ./release/guessanumber.tar.gz  -C ./release/
+cd ./release/tmp/guessanumber/
+gcc guess.c -o game
+
+until [ "$guest" = "n" ]; do
+
+echo "	y) play"
+echo "	n) exit"
+echo " number) number of rounds"
+
+read guest
```

Comment:
The variable name is quite confusing.
I'd suggest to use more meaningful names for variables.